.travis.yml

```diff
@@ -1,68 +1,40 @@
-sudo: true
+os: linux
 language: rust
-cache:
-  directories:
-    - $HOME/.cargo
-    - $HOME/.rustup
-before_cache:
-  - rm -rf /home/travis/.cargo/registry
-rust:
-  - nightly
-  - nightly-2019-06-18
-  - beta
-  - stable
-  - 1.36.0
-matrix:
-    allow_failures:
-        - rust: nightly
-    include:
-      - rust: nightly-2019-06-18
-        before_script:
-          - rustup component add clippy
-        script: cargo clippy
-before_script:
-  - |
-    pip install git+git://github.com/kbknapp/travis-cargo.git --user &&
-    export PATH=$HOME/.local/bin:$PATH
-  - |
-    if [[ "$TRAVIS_RUST_VERSION" == "1.13.0" ]]; then
-        echo "Old Rust detected, removing version-sync dependency"
-        sed -i "/^version-sync =/d" Cargo.toml
-        rm "tests/version-numbers.rs"
-    fi
+cache: cargo
+rust: stable
+jobs:
+  allow_failures:
+    - rust: nightly
+  fast_finish: true
+  include:
+    - os: osx
+    - {}
```

Comment:
What's that?

Reply:
the `- {}` thing

Reply:
It runs stable on Linux. Without this line, it doesn't run it